events/oclcCirculatingEvent.json

```diff
@@ -0,0 +1,10 @@
+{
+  "path": "/search/o1295014271?fbclid=IwAR3iQaNePz2pd1JgoefSEgEbc9mAHnZiXXqY3BfeKp-kTui5lrlT_NDsSfE",
```

Comment:
Did a Facebook tracking id make it into this test event?

Reply:
Fixed